docs/VPN_README.md

```diff
@@ -13,16 +13,62 @@ without help from Continu. To this end we also have a simple [WireGuard](https:/
+WireGuard](https://www.wireguard.com/install/). 
+
+Setting up WireGuard for a user consists of the following steps
+
+1. Select a unique IP address for the user
+2. Have the user generate a public/private key pair
+3. Install the user on the server
+4. Have the user create a client configuration
+5. Have the user test their connection to the server
+
+How the user does some of these steps depends on whether they are using the command-line or GUI version of WireGuard. 
+
+## 1. Select a unique IP address for the user
+
+NOTE: Steps 1 and 2 can take place in parallel. 
+
+Figure out what the next free IP address is by looking at [the server config file](../ansible/templates/wg0.conf). Note that each user has a different number for the `AllowedIPs` line between `100.65.0.` and `/32`. Look at the number for the last user in the list, and increment that number to select the user's IP address. Send the user this IP address so they can create their configuration. 
+
+## 2. Have the user generate a public/private key pair
+
+How they do this depends on whether they are using the command-line or GUI version of WireGuard. Once this is done, have the user send their public key to you so you can install them on the server. 
+
+### Command-line
+
+To generate keys with the command-line, see [the WireGuard docs](https://www.wireguard.com/quickstart/#key-generation). 
+
+### GUI (Mac or Windows)
+
+To generate keys with the GUI, create a new tunnel by clicking on the `+` on the lower left of the GUI and select `Add Empty Tunnel...`. You will see something like the following. Give the tunnel a name (e.g. MVP), and the save. 
 
-## Windows Client
-Once installed, you can add a new tunnel and create public/private keys by creating a new tunnel. 
 ![image](https://user-images.githubusercontent.com/311063/85208559-61b9a500-b2e6-11ea-9c2f-cea6515cb8ff.png)
 
-Once keys are
-generated users then generate a file like:
+## 3. Install the user on the server
+
+NOTE: Steps 3 and 4 can take place in parallel. 
+
+To install a user on the server, edit [the server config file](../ansible/templates/wg0.conf) adding a new `[Peer]` section adding the user's public key and the unique IP address they were assigned in step 1 above. Be sure to add a comment indicating what actual human the public key
+belongs to. Finally, to apply the changed config use Ansible:
+
+```
+ansible-playbook -i inventory_wg.yml -l master --tags=wg_config --ask-vault-pass playbook.yml
+```
+
+You can find the password for the Ansible vault in the `continu-k8.txt` file in our secrets bucket
+(`passwords.mvpstudio.org` bucket on GCE).
+
+Note that `inventory_wg.yml` is the same as `inventory.yml` but the IP addresses are the addresses that show up in the
+WireGuard tunnel. If you are on the OpenVpn tunnel use `inventory.yml` instead.
+
+Finally, create a pull request to check in the modified `wg0.conf` file.
+
+The master server now has IP address 100.64.0.1 on the WireGuard network.
```

Comment:
The master server has always had that address. Adding a new peer doesn't change the server's IP. Maybe reword like:

```suggestion
The user can now access the master server at 100.64.0.1 on the WireGuard network.
```